js/react/components/home/PartnerCarousel.jsx

```diff
@@ -0,0 +1,51 @@
+import { Carousel } from '@mantine/carousel';
+import { Image, MantineProvider } from '@mantine/core';
+import Autoplay from 'embla-carousel-autoplay';
+import { useRef } from 'react';
+
+import { PARTNER_CAROUSEL_DATA } from '../../utils/constants';
+
+function PartnerCarousel() {
+	const autoplay = useRef(Autoplay({ delay: 2500 }));
+
+	const partnerCarouselSlides = PARTNER_CAROUSEL_DATA.map((data) => (
+		<Carousel.Slide
+			key={data.img}
+			className="d-flex flex-column justify-content-center align-items-center"
+		>
+			<a
+				href={data.url ? data.url : '#'}
+				target={data.url ? '_blank' : undefined}
+				rel={data.url ? 'noreferrer' : undefined}
+			>
+				<Image src={data.img} fit="cover" w={150} h="auto" alt={data.alt} />
+			</a>
+		</Carousel.Slide>
+	));
+
+	return (
+		<MantineProvider>
+			<div className="container my-5">
```

Comment:
you can use the `Container` component from mantine here although I'm not sure why we need a container if the carousel should span across the whole screen